docs/sources/shared/upgrade/upgrade-common-tasks.md

```diff
@@ -8,13 +8,13 @@ title: Upgrade guide common tasks
 
 ## Upgrade Grafana
 
-The following sections provide instructions for how to upgrade Grafana based on your installation method.
+The following sections provide instructions for how to upgrade Grafana based on your installation method. For more information on where to find configuration files, refer to [Configuration file location](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/#configuration-file-location).
 
 ### Debian
 
 To upgrade Grafana installed from a Debian package (`.deb`), complete the following steps:
 
-1. In your current installation of Grafana, save your custom configuration changes to a file named `<grafana_install_dir>/conf/custom.ini`.
+1. In your current installation of Grafana, save your custom configuration changes to a file named `<grafana_install_dir>/grafana.ini`.
```

Comment:
```suggestion
1. In your current installation of Grafana, save your custom configuration changes to a file named `<grafana_install_dir>/custom.ini`.
```

Reply:
👀  I understood this like Isabel says.

Reply:
After doing some research, @imatwawana you pointed it correctly that there is NO `custom.ini` in Linux. So in short its ONLY `grafana.ini` in Linux distributions

Kindly, make changes where needed in docs

Thank you again !! :slightly_smiling_face:

Reply:
The other instructions all follow this pattern, so they're fine. I'll go ahead and merge this! Thanks for all your help!